### DISCUSSION TITLE
using magentic web crawler agent - legal considerations

### DISCUSSION THREAD

**Discussion by ssrikantan** [microsoft/autogen#4244] [Q&A]
Hello,
I have been playing around with Magentic and the web crawler agent, and able to access web sites, perform click actions and retrieve the results. While this is very useful, I wanted to know if there are any legal considerations in performing this, and when we need to recommend this as a solution option to our customers.
Looking for some guidance here on that

**Comment by arvindmepani** [comment]
Is Magentic one similar to what it is explained by here before this announcement or it's something different?
[AutoGroq - building AI agents](
https://github.com/jgravelle/AutoGroq)
